catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ExportCommand.java

```diff
@@ -214,6 +227,29 @@ protected Object executeWithSubject() throws Exception {
         return null;
     }
 
+    private StringBuilder getUserInputModifiable() throws IOException {
+        int in;
+        StringBuilder builder = new StringBuilder();
+        while ((in = session.getKeyboard()
+                .read()) != '\r') {
```

Comment:
@ahoffer I will ensure it works on all OSes (though I did test on windows and i believe it worked)
@coyotesqrl Indeed, we must do this if we want to have an "editable" input field. Originally did have it simple like you had above, though if somone backspaces to correct an error for some reason the karaf shell/console does not support this by default making it uneditable. So this implementation allows editing of the input field (basically backspace support)